index.html

```diff
@@ -1005,55 +1005,84 @@ <h2>Continue Watching & Play Next</h2>
         </code>
       </pre>
     </div>
+  </section>
+
+  <section id="play-next-tv-episodes">
+    <h2>Play Next and TV Episodes</h2>
 
     <p>
-      A user agent might want to display the next episode in a TV series to play
-      next. In this case, the user agent should calculate the
-      <dfn>play next episode</dfn> using the following algorithm:
+      A <a>media feed item</a> that is a TV series can have up to two TV
+      episodes embeded in it. These can be embedded directly in the
+      <a>media feed item</a> or embedded within a <a>media item season</a>.
+      If there is more than two episodes they will be ignored.
     </p>
 
     <p>
+      The user agent MUST use the following steps to calculate the
```

Comment:
I assume the next episode would have a "PotentialActionStatus." Couldn't we instead require that if there are two episodes, only one of them is completed/active status and the other is potential? We would then not need to sort or do math with the number of episodes.

My understanding is that the only reason the provider would supply us two episodes is if they intend us to use one for play next.

Reply:
See above

Reply:
It seems like we're going to a lot of effort to stay flexible for a feature we're not sure we'll have (multiple episodes and a picker). Couldn't we make a new version of the spec if we wanted to add a picker? I suppose it could get annoying to support multiple versions, which would explain your preference to leave some flexibility.

Reply:
We don't really do versions of specs. If we were to update the to add a picker in the future we would update this spec and seamless backwards compatibility is important. That way a website that is using the picker and returning multiple tv eps would still work in an old version of Chrome that has the old API.

Reply:
I see. Thanks for the explanation.

Apologies, I had a couple other comments I left below that I forgot until now. Otherwise LGTM.